cockatrice/src/window_main.cpp

```diff
@@ -60,7 +60,7 @@
 #define GITHUB_PAGES_URL "https://cockatrice.github.io"
 #define GITHUB_CONTRIBUTORS_URL "https://github.com/Cockatrice/Cockatrice/graphs/contributors?type=c"
 #define GITHUB_CONTRIBUTE_URL "https://github.com/Cockatrice/Cockatrice#cockatrice"
-#define GITHUB_TRANSLATOR_RECOGNIZE_URL "https://github.com/Cockatrice/Cockatrice/wiki/Translators"
+#define GITHUB_TRANSIFEX_TRANSLATORS_URL "https://github.com/Cockatrice/Cockatrice/wiki/Translator-Hall-of-Fame"
```

Comment:
That's the original working link. We just have the [Translators Hall of Fame](https://github.com/Cockatrice/Cockatrice/wiki/Translator-Hall-of-Fame) to name them. It has always been like that. Zach created it like one year ago.

I'm not sure if I like that page name though...

Maybe if was broken all the time? Or Can't tell for sure.